src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -141,7 +143,20 @@ public int getCentralTagId() {
         // make a temp holder var for least Y translation, set to first tags translation
         double minY = 1.0e6; // big number
         int targetID = -1;
-        for (PhotonTrackedTarget tag : targetResult.getTargets()) { // for every target in camera
+        for (PhotonTrackedTarget tag : targetResult.getTargets()) { // for every target in camera            
+            // find id for current tag we are focusing on
+            int tempTagID = tag.getFiducialId();
+
```

Comment:
Can you add a test for legal IDs?  tempTagId >=1 and <= 8.  I have seen log messages about "tag 20" which is caused just by noise. We should filter as tightly as we can.